mix.exs

```diff
@@ -53,7 +53,10 @@ defmodule EpochElixir.Mixfile do
       {:aewallet, github: "aeternity/elixir-wallet"},
       {:msgpax, "~> 2.1.1"},
       {:bip0173, "~> 0.1.2"},
-      {:erl_base58, "~> 0.0.1"}
+      {:erl_base58, "~> 0.0.1"},
+      {:merkle_patricia_tree,
+       git: "https://github.com/meivantodorov/merkle_patricia_tree.git",
+       branch: "meivantodorov/deleting-node"}
```

Comment:
can you add this in master branch, then I will for it to aeternity

Reply:
It is still in progress, after it is finished You will fork it.

Reply:
ok, ping me once its done